shared/connection.go

```
@@ -169,13 +169,43 @@ func (c *Connection) Exec(command string, args ...string) ([]byte, error) {
 	return utils.RunCmdOutput(zerolog.DebugLevel, cmd, cmdArgs...)
 }
 
+// WaitForContainer waits up to 10 sec for the container to appear.
+func (c *Connection) WaitForContainer() error {
+	for i := 0; i < 10; i++ {
+		podName, err := c.GetPodName()
+		if err != nil {
+			log.Warn().Err(err)
+			time.Sleep(1 * time.Second)
+			continue
+		}
+		args := []string{"exec", podName}
+		command, err := c.GetCommand()
+		if err != nil {
+			log.Fatal().Err(err)
```

better return the error rather than `log.Fatal()`